### ISSUE TITLE
Can't get to bitbucket from Chicoma

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/compass#454]
```
$ git submodule update --init
Cloning into '/usr/projects/climate/xylar/compass/add_chicoma-cpu/E3SM-Project/components/mpas-ocean/src/SHTNS'...
fatal: unable to access 'https://bitbucket.org/nschaeff/shtns.git/': Received HTTP code 403 from proxy after CONNECT
fatal: clone of 'https://bitbucket.org/nschaeff/shtns.git' into submodule path '/usr/projects/climate/xylar/compass/add_chicoma-cpu/E3SM-Project/components/mpas-ocean/src/SHTNS' failed
Failed to clone 'components/mpas-ocean/src/SHTNS'. Retry scheduled
Cloning into '/usr/projects/climate/xylar/compass/add_chicoma-cpu/E3SM-Project/components/mpas-ocean/src/SHTNS'...
fatal: unable to access 'https://bitbucket.org/nschaeff/shtns.git/': Received HTTP code 403 from proxy after CONNECT
fatal: clone of 'https://bitbucket.org/nschaeff/shtns.git' into submodule path '/usr/projects/climate/xylar/compass/add_chicoma-cpu/E3SM-Project/components/mpas-ocean/src/SHTNS' failed
Failed to clone 'components/mpas-ocean/src/SHTNS' a second time, aborting
```

**Comment by xylar** [comment]
@sbrus89, I'm not able to get to Bitbucket when I'm on Chicoma at LANL.  I realize this is an odd issue but it means I can't build standalone MPAS-Ocean on that machine.

I'm wondering if we could create our own fork of SHTNS in E3SM-Project and use that for the submodule instead.

**Comment by jonbob** [comment]
@xylar - I was having the same problem this past few days and wondering if it will be different from inside the LANL network. Regardless, we need to be able to clone E3SM on the IC machines from anywhere, so please let me know if I can help.

**Comment by xylar** [comment]
@jonbob, I haven't seen this problem in my earlier testing so I think it may be that LANL changed the policy.  But given that we aren't getting any other submodules from bitbucket, I think making our own copy is the safest fix.

**Comment by xylar** [comment]
I made my own fork (following https://gist.github.com/mandiwise/5954bbb2e95c011885ff):
https://github.com/xylar/shtns
I seem to be able to proceed after updating the URL as described here:
https://stackoverflow.com/a/10318907

**Comment by jonbob** [comment]
@xylar -- I agree completely. And you're right, this is new behavior so some LANL IC policy must have changed

**Comment by xylar** [comment]
This is probably a low priority for compass because it still is performing very badly on Chicoma and I don't have time to debug any further but E3SM will also be affected.

**Comment by sbrus89** [comment]
@xylar - Sorry about this! I agree that having forks under E3SM-Project is the best way to fix the issue. I believe that @rljacob has helped me with getting those set up in the past.

**Comment by rljacob** [comment]
Yes I could add that repo to E3SM-Project.   Is all of bitbucket.org blocked from that machine?  Can you ping it?

**Comment by xylar** [comment]
@rljacob, no, I'm not able to ping bigbucket.org but I'm also not able to ping github.com so I don't think that's very informative.  I think ping might just be blocked.

**Comment by xylar** [comment]
I believe this has been resolved.  Perhaps Chicoma now can get to BitBucket.
